Assets/HoloToolkit-Examples/UX/Scripts/InteractiveGroup.cs

```diff
@@ -7,17 +7,18 @@ namespace HoloToolkit.Examples.InteractiveElements
     [RequireComponent(typeof(InteractiveSet))]
     public class InteractiveGroup : MonoBehaviour
     {
+        [Tooltip("Gameobject containing GridLayoutGroup")]
+        public GameObject grid;
+
         [Tooltip("Prefab for your interactive.")]
         public GameObject InteractivePrefab;
 
+        [Tooltip("scale for new instance of InteractivePrefab")]
+        public Vector3 prefabScale = new Vector3(2000, 2000, 2000);
+
         [Tooltip("Data to fill the InteractiveSet.")]
         public List<string> Titles = new List<string>();
 
-        [Tooltip("number of elements after which we start a new column")]
-        public int Rows = 3;
```

Comment:
Is there no way/need to specify a row count anymore?

This seems like a possible breaking change.

Reply:
yes, you still can, in fact it is way better now - you can specify max rows or colums in the inspector for the grid.

Reply:
Great! Just checked it out and it looks super handy!